includes/options/class-amp-options-manager.php

```diff
@@ -353,12 +351,15 @@ public static function render_welcome_notice() {
 		}
 
 		?>
-		<div class="notice notice-info is-dismissible" id="<?php echo esc_attr( $notice_id ); ?>">
-			<img src="<?php echo esc_url( amp_get_asset_url( 'images/amp-logo-icon.svg' ) ); ?>" alt="AMP" width="40" height="40" style="float: left; margin: 5px;">
-			<h1><?php esc_html_e( 'Welcome to the AMP for WordPress plugin v1.0', 'amp' ); ?></h1>
-			<p><?php esc_html_e( 'Thank you for installing! Bring the speed and features of the open source AMP project to your site, the WordPress way, complete with the tools to support content authoring and website development.', 'amp' ); ?></p>
-			<h2><?php esc_html_e( 'What&#8217;s New', 'amp' ); ?></h2>
-			<p><?php esc_html_e( 'From granular controls that help you create AMP content, to Core Gutenberg support, to a sanitizer that only shows visitors error-free pages, to a full error workflow for developers, this v1.0 release makes it easier than ever to bring a rich, performant experience to your WordPress site via AMP HTML.', 'amp' ); ?></p>
+		<div class="amp-welcome-notice notice notice-info is-dismissible" id="<?php echo esc_attr( $notice_id ); ?>">
+			<div class="notice-dismiss"></div>
+			<div class="amp-welcome-icon-holder">
+				<img class="amp-welcome-icon" src="<?php echo esc_url( amp_get_asset_url( 'images/amp-welcome-icon.svg' ) ); ?>" srcset="<?php echo esc_url( amp_get_asset_url( 'images/amp-welcome-icon.svg' ) ); ?>" />
```

Comment:
Why `srcset`?

Reply:
Leftover from a thought I had about adding a `.png` fallback. Thanks for catching this!